### DISCUSSION TITLE
关于Python优先级的问题

### DISCUSSION THREAD

**Discussion by geekroscom** [MiRoboticsLab/cyberdog_ros2#9] [General]
目前铁蛋的Python3优先级高于Python2，导致一些默认需要Python2编译、运行的程序无法使用，调整优先级后，导致错误：Illegal instruction (core dumped)，希望新版固件能够恢复Python2为最高优先级，其他需要使用Python3的地方直接：python3 ******.py

**Comment by homalozoa** [comment]
哈喽，python2目前已经停止维护（2020年），所以默认是python3，请避免使用python2的库，尽量适配到python3
